Integration Account Partner Metadata

emielkool opened the pull request:
It is now in the UI instead of only JSON editor, screenshot also changed.

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

emielkool committed:
Update logic-apps-enterprise-integration-metadata.md

MetaData can be edited in the GUI instead of the JSON definition of the Partner. Changed the text but screenshot also needs to be updated.

emielkool committed:
Add files via upload
